frontend/src/pages/StartPage.tsx

```diff
@@ -1,9 +1,16 @@
-import NavBar from "../component/NavBar";
+import StockCard from "../component/StockCard";
+import {Stock} from "../model/Stock";
+
+export type StartPageProps = {
+    stocks : Stock[]
+}
+
+export default function StartPage({stocks} : StartPageProps) {
+
 
-export default function StartPage() {
     return (
-        <div>
-            <NavBar />
-        </div>
+            <div>
+                {stocks.map(stock => <StockCard stock={stock}/>)}
```

Comment:
Du brauchst einen eindeutigen Key wenn du eine Componente in einer Liste anzeigst
`<StockCard key={stock.id} stock={stock} />`